<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YiPlusMain</class>
 <widget class="QWidget" name="YiPlusMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="Edit_Password">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GCN0kn9VEM5pc7zPakk2YA==</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Time_EndRob">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_LoginTest">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_ImportAccount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>从txt导入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所有的用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>抢购结束时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日志信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>抢购开始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_UserName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>15002326234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_StartRob">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>立即开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_AddAccount">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加账户</string>
   </property>
  </widget>
  <widget class="QListView" name="List_Logs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>761</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Time_StartRob">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="Table_AllAccount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>761</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_StopAll">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_ClearLog">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>315</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除日志</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckBox_useProxy">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>使用代理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>98</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>抢购时间范围</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_startTime">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_stopTime">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_setTimeRangeAndRob">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置范围并开启</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckBox_PrintLog">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>打印日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_EnableAccount">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>不启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_DelAccount">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
